core-bundle/src/Composer/ScriptHandler.php

```diff
@@ -70,6 +70,11 @@ private static function executeCommand(array $cmd, Event $event): void
             $command[] = $verbose;
         }
 
+        // Backwards compatibility with symfony/process <3.3 (see #1964)
+        if (method_exists(Process::class, 'setCommandline')) {
+            $command = implode(' ', $command);
```

Comment:
Isn’t it a bit risky to just implode the command line parts without escaping?

```suggestion
            $command = implode(' ', array_map('escapeshellarg', $command));
```

Reply:
See #2026